internal/zinc-core/src/main/scala/sbt/internal/inc/Relations.scala

```diff
@@ -139,7 +146,7 @@ trait Relations {
    *
    * This reflects Scala's compiler behavior of generating two class files per top level object declaration.
    */
-  def binaryClassName: Relation[String, String]
+  def productClassName: Relation[String, String]
```

Comment:
Even within the context of `Relations` alone the `binaryClassName` above is inconsistent with the use of the term binary in `binaryDeps` or "binary dependencies", correct?